tests/vitest-setup.ts

```diff
@@ -0,0 +1,7 @@
+import '@testing-library/jest-dom/vitest';
+import {cleanup} from '@testing-library/react'
+import {afterEach} from 'vitest'
+
+console.log("vitest-setup.ts loaded")
```

Comment:
Should we keep the log?

Reply:
Personally i like having that log to see if it **really** loads the config files 😅 But i can remove them if we are done moving parts around